makefile

```diff
@@ -1,4 +1,33 @@
-clients: .PHONY
-	ruby -rjson -r./utils/clients -e 'Clients.check(JSON.parse(File.read("clients.json"), symbolize_names: true))'
+MD_FILES:=$(shell find {commands,topics} -name '*.md')
+TEXT_FILES:=$(patsubst %.md,tmp/%.txt,$(MD_FILES))
+SPELL_FILES:=$(patsubst %.txt,%.spell,$(TEXT_FILES))
 
-.PHONY:
+spell: tmp/commands tmp/topics $(SPELL_FILES)
+	find tmp -name '*.spell' | xargs cat > tmp/all.spell
+	cat tmp/all.spell
+	test -s tmp/all.spell && exit 1
+
+$(TEXT_FILES): tmp/%.txt: %.md
+	./bin/text $< > $@
+
+$(SPELL_FILES): %.spell: %.txt tmp/dict
+	aspell -a --extra-dicts=./tmp/dict 2>/dev/null < $< | \
```

Comment:
We should pass `--lang=en` here. Not everyone has english as the default, and this should work on user machines as well.